lib/ui/painting/display_list_deferred_image_gpu.cc

```diff
@@ -67,4 +111,38 @@ std::optional<std::string> DlDeferredImageGPU::get_error() const {
   return error_;
 }
 
+DlDeferredImageGPU::ImageWrapper::ImageWrapper(
+    const GrBackendTexture& texture,
+    sk_sp<GrDirectContext> context,
+    fml::RefPtr<fml::TaskRunner> raster_task_runner,
+    fml::RefPtr<SkiaUnrefQueue> unref_queue)
+    : texture_(texture),
+      context_(std::move(context)),
+      raster_task_runner_(std::move(raster_task_runner)),
+      unref_queue_(std::move(unref_queue)) {}
+
+void DlDeferredImageGPU::ImageWrapper::OnGrContextDestroyed() {
+  FML_DCHECK(raster_task_runner_->RunsTasksOnCurrentThread());
+
+  if (texture_.isValid()) {
+    unref_queue_->DeleteTexture(std::move(texture_));
```

Comment:
Reset `texture_` to an invalid `GrBackendTexture`
(AFAICT `GrBackendTexture` does not have a move constructor that will invalidate `texture_`)

Also drop the `context_` reference to the `GrDirectContext`

Reply:
Done